config/initializers/octokit.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Octokit.configure do |config|
+  config.access_token = ENV['GITHUB_ACCESS_TOKEN']
```

Comment:
```suggestion
  config.access_token = ENV.fetch 'GITHUB_ACCESS_TOKEN'
```

Should we raise if this isn't set so we know why github integration isn't working?